src/darwin/Framework/CHIP/MTRLogging_Internal.h

```diff
@@ -22,5 +22,5 @@
 
 #define MTR_LOG_ERROR(msg, ...) ChipLogError(NotSpecified, msg, ##__VA_ARGS__)
 #define MTR_LOG_DEFAULT(msg, ...) ChipLogProgress(NotSpecified, msg, ##__VA_ARGS__)
-#define MTR_LOG_INFO(msg, ...) ChipLogDetail(NotSpecified, msg, ##__VA_ARGS__)
+#define MTR_LOG_INFO(msg, ...) ChipLogProgress(NotSpecified, msg, ##__VA_ARGS__)
```

Comment:
Thing is, these log levels are _supposed_ to correspond to the error/default/info/debug values of `os_log_type_t` and have the same semantics.

What we should probably do is remove uses of MTR_LOG_INFO (and maybe the macro?), and replace with default/debug, if we think it's being misused.

Note that right now MTR_LOG_DEBUG in fact ends up using OS_LOG_TYPE_INFO (just like MTR_LOG_INFO does), but that's because it's being squeezed through the ChipLog* bits that have a different number of levels.

In particular, I am not sure whether we want the "got attribute report" logs in MTRDevice to be TYPE_INFO or TYPE_DEFAULT; those can be pretty spammy.

That said, I do agree that a bunch of stuff we log as INFO should have been DEFAULT; one option is to take this PR for now (with the problems above fixed), but then do a followup to get rid of MTR_LOG_INFO and audit all the remaining callers....